builtin/providers/aws/resource_aws_route_table_test.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/awslabs/aws-sdk-go/aws"
 	"github.com/awslabs/aws-sdk-go/service/ec2"
+	hashiec2 "github.com/hashicorp/aws-sdk-go/gen/ec2"
```

Comment:
I had to pull in the old EC2 library because the VPN gateway code isn’t ported over to the new library yet.